packs/feat-effects/stance-talon-stance.json

```diff
@@ -1,6 +1,6 @@
 {
     "_id": "cvIwUEempg0cfhFB",
-    "img": "systems/pf2e/icons/unarmed-attacks/foot.webp",
+    "img": "systems/pf2e/icons/unarmed-attacks/talon.webp",
```

Comment:
```suggestion
    "img": "icons/creatures/claws/claw-talons-glowing-orange.webp",
```

Reply:
Those are the same images, just one is provided by PF2e and one is provided by Foundry VTT. Figured we would want to use the one provided by PF2e since that one is used for Talon strikes, otherwise why does PF2e include it's own talon image.

Reply:
The system ones tend to be higher-resolution. We're moving away from the PF2e images where there are duplicates, but can't actually delete the old files without migration.